aidants_connect_web/tests/test_views/test_service.py

```diff
@@ -171,3 +171,14 @@ def test_guide_utilisation_url_triggers_the_correct_view(self):
     def test_stats_url_triggers_the_correct_template(self):
         response = self.client.get("/guide_utilisation/")
         self.assertTemplateUsed(response, "aidants_connect_web/guide_utilisation.html")
+
+
+@tag("service")
+class FAQTests(TestCase):
+    def test_guide_utilisation_url_triggers_the_correct_view(self):
+        found = resolve("/faq/")
+        self.assertEqual(found.func, service.faq)
+
+    def test_stats_url_triggers_the_correct_template(self):
```

Comment:
```suggestion
    def test_faq_url_triggers_the_correct_template(self):
```

bon après je sais que ça sera un jour refactoré cette histoire :)